### PR TITLE
Sample of form template for Create Listing with Jinja

### PR DESCRIPTION
**Pull request by vyshor**
1. Mainpage uses the data from the listing, dynamic data rather than static
2. Change create listing to make use of Jinja template too

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -83,5 +95,5 @@ def handle_listings():
     # the "static" directory. See:
     # http://flask.pocoo.org/docs/1.0/quickstart/#static-files. Once deployed,
     # App Engine itself will serve those files as configured in app.yaml.
-    app.run(host='127.0.0.1', port=8080, debug=True)
+    app.run(host='127.0.0.1', port=8081, debug=True)
```
Comment (hongshaoyang) [inline main.py]: Oh, why the need to change this port to 8081?

**Review by maggiekoesno — DISMISSED** [review]
Also wondering why the change from port 8080 to 8081.
That aside, LGTM 👍

File: main.py (hunk 1 above)
Comment (vyshor) [reply main.py]: Nothing much, just that sometimes other applications are taking up the port 8080. So had to change it for a quick fix.

Comment (hongshaoyang) [reply main.py]: Ah, I see. Do you want to revert changes to this then?

**Review by hongshaoyang — CHANGES_REQUESTED** [review]
Revert changes to port in main.py

**Review by hongshaoyang — APPROVED** [review]

**Pull request merged**